rs/nns/governance/src/governance.rs

```diff
@@ -5108,6 +5174,25 @@ impl Governance {
             .expect("NetworkEconomics not present")
     }
 
+    pub fn voting_power_economics(&self) -> &VotingPowerEconomics {
```

Comment:
Why return a reference? `VotingPowerEconomics` implements Copy and should be super fast to clone

Reply:
It's true that under its current definition, VotingPowerEconomics is O(1), i.e. small enough to be Copy. However, our future selves would look at it and think, "this is a great place for me to put **_more stuff_**", and they're right: there is no good reason you shouldn't be able to add stuff to VPE. That's what it's there for: to be a place where you add new configuration options. If people started adding big things to VPE, our copying would suddenly become (way) more expensive, and most likely, nobody would notice that degradation before release.

Anyway, yes, big additions is very theoretical; I do not realistically foresee anyone wanting to actually add more stuff to VotingPowerEconomics, but that's what they all say. Failure of imagination is not a reliable reason to (not) do things. Ultimately, VPE does not have a bounded remit. In such cases, I think we should choose reference. It's definitely less convenient sometimes, but in many cases, it's not hard.

I honestly don't know how VotingPowerEconomics is Copy. I thought that in order for Prost to generate a Copy struct from a message, you have to explicitly tell Prost, "trust me, bro: this should be Copy"?? E.g. IIUC, Prost does [not give Neuron Copy][neuron]. I could not find any such configuration -> I am confused why it comes out as Copy.

[neuron]: https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/nns/governance/src/gen/ic_nns_governance.pb.v1.rs?L177-179